From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Anil Kumar <anil.kumar.k@intel.com>
Date: Fri, 13 Sep 2019 13:00:33 -0700
Subject: [PATCH] Revert "[REVERTME] usb: roles: intel_xhci: Set device mode as
 default."

This reverts commit 544b6dce1f4445bb3397d7c2b020b1fd4325ac07.

Change-Id: I105324d7a8647114dd1e59ec88ffb5a121b3160d
Tracked-On: PKT-2619
---
 drivers/usb/roles/intel-xhci-usb-role-switch.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/usb/roles/intel-xhci-usb-role-switch.c b/drivers/usb/roles/intel-xhci-usb-role-switch.c
index 0d72187ee561..9c6606261b2e 100644
--- a/drivers/usb/roles/intel-xhci-usb-role-switch.c
+++ b/drivers/usb/roles/intel-xhci-usb-role-switch.c
@@ -178,10 +178,6 @@ static int intel_xhci_usb_probe(struct platform_device *pdev)
 	pm_runtime_set_active(dev);
 	pm_runtime_enable(dev);
 
-	intel_xhci_usb_set_role(dev, USB_ROLE_HOST);
-	msleep(10);
-	intel_xhci_usb_set_role(dev, USB_ROLE_DEVICE);
-
 	return 0;
 }
 
-- 
https://clearlinux.org

